Aquarius/Src/Aquarius/Events/KeyboardEvent.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "Event.h"
+
+#include <GLFW/glfw3.h>
+
+
+namespace Aquarius {
+
+    class KeyEvent : public Event
+    {
+    public:
+
+        int getCode()
+        {
+            return m_keyCode;
+        }
+
+    protected:
+        int m_keyCode;
+
+        KeyEvent(int keyCode) : m_keyCode(keyCode) {}
+    };
+
+    class KeyPressedEvent : public KeyEvent
+    {
+    public:
+        KeyPressedEvent(int keyCode) : KeyEvent(keyCode) {}
+
+        EVENT_TYPE(KeyPressedEvent)
+
+        void logEvent()
```

Comment:
Looks as tho every subclass of event has a logEvent class, maybe it should be declared virtual in the base event class?